apps/server/src/defichain/controllers/StatsController.ts

```diff
@@ -0,0 +1,15 @@
+import { StatsData } from '@defichain/whale-api-client/dist/api/stats';
```

Comment:
```suggestion
import { stats } from '@defichain/whale-api-client';
```

And then use `stats.StatsData` below. To prevent dipping into `dist` 😛

Reply:
damn auto import